core/src/smartcontracts/isi/triggers/mod.rs

```diff
@@ -130,7 +112,26 @@ pub mod isi {
         }
     }
 
-    impl Execute for Burn<u32, Trigger<TriggeringEventFilterBox>> {
+    impl Execute for Unregister<Trigger> {
```

Comment:
Do you need change order? `Register/Unregister` and then `Mint/Burn` seems reasonable.
I'm also thinking about resolving conflicts.

Reply:
This was unintentional... I'll revert this